Source/WebInspectorUI/UserInterface/Views/Main.css

```diff
@@ -579,6 +578,27 @@ body[dir=rtl] [dir=ltr] .go-to-arrow {
     inset-inline-end: 0.5em;
 }
 
+.user-preferences-content {
+    display: flex;
+    flex-direction: column;
+    padding-inline: 5px;
+}
+
+.user-preferences-content > h5 {
+    margin-block-start: 4px;
+    margin-block-end: 8px;
+    font-size: 11px;
+}
+
+.user-preferences-content label {
```

Comment:
Isn't the `label` a direct child of `.user-preferences-content`?  If so, please use `.user-preferences-content > label`.